bundles/net.tourbook/src/net/tourbook/ui/views/TourBlogView.java

```diff
@@ -277,7 +282,7 @@ private void addTourEventListener() {
             return;
          }
 
-         if ((eventId == TourEventId.TOUR_CHANGED) && (eventData instanceof TourEvent)) {
+         if (((eventId == TourEventId.TOUR_CHANGED) && (eventData instanceof TourEvent))) {
```

Comment:
Why additional brackets ?

Reply:
My bad, I did some debugging and forgot to remove them.
In some other editors like Visual Studio, the color is changed when additional brackets are found but unfortunately, it doesn't seem that Eclipse has that feature

![image](https://github.com/mytourbook/mytourbook/assets/25235665/ddaaa406-581c-454b-8868-1d820569b81d)

Reply:
Fixed